rosa_architecture/rosa-aws-amis.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-aws-ami"]
+= AWS AMIs for {product-title}
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-aws-ami
+
+toc::[]
+
+The following list contains the Amazon Machine Images (AMIs) for {product-title} clusters. For more information, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html#ami-using[Amazon Machine Images (AMI)] in the AWS documentation.
+
+You must use the image version that is the same as the AMI version when you installed your cluster. For example, if you installed the cluster using 4.14, but your cluster is on OpenShift version 4.16, you must still allow for AMIs that use the `rhcos-414-...` because new instances use same AMI version.
```

Comment:
This text should be updated to remove the reference to `rhcos-414-…​` because we removed the image description.